pythoninput.md

```diff
@@ -0,0 +1,64 @@
+Two main methods for taking in input with Python 3:
+***
+
+```python
+input()
+```
+
+Returns line read from input after being converted to a string
+
+Optionally can be provided an argument that is printed to standard ouput
+
+``` python
+>>> a = input("Enter a number\n")
+Enter a number
+5
+>>> print(a)
+'5'
+```
+
+If input is desired in some type other than string, you can convert it like so:
+
+```python
+a = int(input())
+b = float(input())
+```
+
+[input() Documentation](https://docs.python.org/3/library/functions.html#input)
+
+In some cases it may be useful to separate the line of input based on a delimiter 
+such as ' ' or ','
+
+In Python we can do that with:
+
+```python
+split()
+```
+
+Returns a list of words in the strings, separated by the provided delimter. If argument is left blank then
+separates on spaces.
+
+```python
+>>> groceries = "apples, bananas, milk, eggs"
+>>> grocery_list = groceries.split(',')
+>>> print(grocery_list)
+['apples', 'bananas', 'milk', 'eggs']
+```
+
+[split() Documentation](https://docs.python.org/3/library/stdtypes.html#str.split)
+
+***
+
+```python
+from sys import stdin
+```
+Can be used to iterate over lines of input, such as test cases for Kattis problems
```

Comment:
I think it would be worth mentioning this API would be useful when the number of input lines is unknown (read until EOF). The other way of handling the same situation is to catch `EOFError` using `input()`.